tests/network-tests/src/flows/proposals/manageLeaderRole.ts

```diff
@@ -147,6 +150,7 @@ async function manageLeaderRole(api: Api, env: NodeJS.ProcessEnv, group: Working
 
   // Approve new leader reward
   await new FixtureRunner(voteForeLeaderRewardFixture).run()
+  assert(voteForeLeaderRewardFixture.proposalExecuted)
```

Comment:
This assertion is missing for `voteForDecreaseStakeProposal`, `voteForSlashProposalFixture` and `terminateLeaderRoleProposalFixture` and also in `flows/proposals/workingGroupMintCapacityProposal.ts` and `flows/workingGroup/workerPayout.ts`

Reply:
Well spotted. I'll add them for completeness.